tests/unittests/Foundation/NSFileManagerTests.mm

```diff
@@ -219,7 +219,7 @@
 
     NSDictionary* attributes = @{
         NSFileType : NSFileTypeRegular,
-        NSFileImmutable : @YES,
```

Comment:
This test becomes a no-op if you remove this. Specifically, the default state of a "regular file" is `regular file` + `!immutable`.